src/guides/v2.3/frontend-dev-guide/theme-best-practice.md

```diff
@@ -30,5 +30,10 @@ We recommend using the following best practices when developing themes:
    ```xml
    <a href="#"><?= __('Click to download'); ?></a>
    ```
+1. By following the standard conventions, you give your code a professional look while making it easier to read. Ensure that your Magento coding standards are based on PSR coding standards.
```

Comment:
> Ensure that your Magento coding standards are based on PSR coding standards.

Please extend this sentence because Magento’s Coding Standards are based on the [Zend Coding standards](http://framework.zend.com/manual/1.12/en/coding-standard.html) and [PSR2](http://www.php-fig.org/psr/psr-2/), [PSR4](http://www.php-fig.org/psr/psr-4/).